### PR TITLE
feat(docs): Add Podman Compose guide

### PR DESCRIPTION
**Pull request by wrussell1999**
Guide following the same format as the Docker Compose guide. Originally tested by Will and Paul.

Closes https://github.com/kestra-io/docs/issues/1062

### CODE REVIEW

File: content/docs/02.installation/14.podman.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Podman Compose
+icon: /docs/icons/podman.svg
```
Comment (shrutimantri) [inline content/docs/02.installation/14.podman.md:3]: This icon does not exist in the repo. Can you add it?

Comment (wrussell1999) [reply content/docs/02.installation/14.podman.md:3]: Added!

File: public/docs/icons/podman.svg
Comment (shrutimantri) [inline public/docs/icons/podman.svg:1]: This wont be visible on a black background.

Comment (wrussell1999) [reply public/docs/icons/podman.svg:1]: Fixed

File: content/docs/02.installation/14.podman.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Launch Kestra
+
+Use the following command to create a Podman machine, start it up and launch Kestra on it:
+
+```bash
+podman machine init --cpus 2 --rootful -v /tmp:/tmp -v $PWD:$PWD
+podman machine start
+podman compose up -d
+```
+
```
Comment (paulgrainger85) [inline content/docs/02.installation/14.podman.md:38]:
```suggestion
Note podman executes containers through a VM on your local machine. In order to access local volumes from your container you need to ensure you mount these to the podman VM, hence the `-v /tmp:/tmp -v $PWD:$PWD` arguments.  
```

**Review by paulgrainger85 — CHANGES_REQUESTED** [review]
wdyt - adding some additional context on why you need to mount those volumes

**Pull request merged**